src/components/audio-player/audio-player.js

```diff
@@ -10,6 +10,7 @@ const defaultOptions = {
 
 class AudioPlayer {
 	constructor({wrapperEl, options={}}) {
+		this.allAudioEls              = document.querySelectorAll("audio");
```

Comment:
i don't think this needs to be a property of the audio player object.  it's also fragile to set it in the constructor because what if players are dynamically added or removed to the page later?  you'll get errors on the ones that were removed and it won't run for the ones that were added.